### PR TITLE
added layout notification

### PR DESCRIPTION
**Pull request by handrok**

### CODE REVIEW

File: src/engraving/rendering/dev/pagelayout.h [hunk 1]
```diff
@@ -24,6 +24,8 @@
 
 #include "layoutcontext.h"
 
+#include <global/modularity/ioc.h>
```
Comment (RomanPudashkin) [inline src/engraving/rendering/dev/pagelayout.h:27]: Unused. Also, we only use <> for either system headers (std, platform API, etc), or Qt

File: src/engraving/rendering/dev/pagelayout.cpp [hunk 2]
```diff
@@ -225,6 +226,10 @@ void PageLayout::collectPage(LayoutContext& ctx)
                 collected = true;
             }
         }
+
+        float curPercent = static_cast<float>(ctx.state().measureNo()) / ctx.dom().lastSegment()->measure()->index();
```
Comment (RomanPudashkin) [inline src/engraving/rendering/dev/pagelayout.cpp:230]: Worth checking that lastSegment() is not null

**Comment by RomanPudashkin** [comment]
Please rebase (there should only be 1 commit)

File: src/engraving/rendering/dev/pagelayout.h (hunk 1 above)
Comment (handrok) [reply src/engraving/rendering/dev/pagelayout.h]: fixed

File: src/engraving/rendering/dev/pagelayout.cpp (hunk 2 above)
Comment (handrok) [reply src/engraving/rendering/dev/pagelayout.cpp]: fixed

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**